lib/et_fake_ccd/version.rb

```diff
@@ -1,3 +1,3 @@
 module EtFakeCcd
-  VERSION = "3.0.0"
+  VERSION = "3.0.1"
```

Comment:
This is out of sync with your Gemfile.lock - you must always run bundle install after changing this